src/components/definition-list/dd.component.js

```diff
@@ -5,11 +5,13 @@ import { StyledDd } from './definition-list.style';
 
 const Dd = ({
   mb = 2,
-  children
+  children,
+  align = 'right'
 }) => {
   return (
     <StyledDd
       mb={ mb }
+      align={ align }
```

Comment:
This doesn't appear to be used anywhere. Please remove.

Reply:
Consider it done